docs/content/overview/product-telemetry.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Product Telemetry"
+description: "Learn how and why NGINX Ingress Controller collects product telemetry data."
+weight: 500
+toc: true
+---
+
+## Overview
+
+NGINX Ingress Controller collects product telemetry data to allow its developers to understand how it's deployed and configured by users.
+The goal is to use this data to triage development work and prioritize features and functionality that will provide the most value to our users.
+
+## About Product Telemetry
+
+Product telemetry is collected by default.
+Data is collected once every 24 hours, and is sent to a service managed by F5 over HTTPS.
+Personally identifiable information (PII) is **not** collected. 
+
+> [!NOTE]  
+> If you prefer to not have data collected, you can [opt-out](#opt-out) when installing NGINX Ingress Controller.
+
+## Data Collected
+
+Below is a list of current set data points that are collected and reported by the NGINX Ingress Controller:
```

Comment:
```suggestion
These are the data points collected and reported by the NGINX Ingress Controller:
```
It's self-evident as a list, so there's no need to mention it's a list or announce a list is about to be shown when the reader is going top down anyway.

"current set data points" is an awkward term. Data points are implicitly precise, and even if there is the possibility of additional points in the future, mentioning it could put people off. If there's no plans, there's no need for the specificity of "current".